### ISSUE TITLE
display thunderbolt issue

### ISSUE THREAD

**Issue by orbee404** [doanhmaple/Razer-Blade-15-Advanced-2018-Hackintosh#9]
Hey there, just a questione. Do you know if the thunderbolt port is linked to the nvidia gpu or is it possibile to use it on macos?

**Comment by mendokusai-931** [comment]
The USB C port (Thunderbolt) is connected to the Intel UHD 630 so it's possible to use it on macOS, DisplayPort, and HDMI is connected to NVIDIA GPU.

**Comment by orbee404** [comment]
> 
umh on windows if i link the usb c adapter to the thunderbolt port on the nvidia control panel it shows like is linked to the gpu, so i think is linked to the nvidia.

**Comment by mendokusai-931** [comment]
Try to disable your Nvidia GPU in device manager the plug in a display through USB C, if it's still run it means that the port is connected to your Intel iGPU. I did it myself and my port is connected to my iGPU.

**Comment by orbee404** [comment]
Ok, i will try thanks

**Comment by darkloki19** [comment]
On the Mac Side, the display port through USB C should work fine right? There is nothing additional that I need to do or that is required right?

**Comment by orbee404** [comment]
yep but my razer 2019 (i7 9750h, Nvidia 1660ti) the Thunderbolt/USB c is
linked to the dgpu so no external monitor on Mac
